ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -253,58 +245,45 @@ class AlertsView extends React.Component {
               make changes
             </Alert>
           )}
-          <AlertsViewControls
-            dropdownOptions={id ? [] : alertDropdowns}
-            alertSummaries={alertSummaries}
-            issueTrackers={issueTrackers}
-            optionCollectionMap={optionCollectionMap}
-            fetchAlertSummaries={id => this.fetchAlertSummaries(id, true)}
-            updateViewState={state => this.setState(state)}
-            bugTemplate={bugTemplate}
-            user={user}
-            {...this.props}
-          />
-          {pageNums.length > 1 && (
-            <Row className="justify-content-center pb-5">
-              {/* The first and last pagination navigation links
-              aren't working correctly (icons aren't visible)
-              so they haven't been added */}
-              <Pagination aria-label={`Page ${page}`}>
-                {page > 1 && (
-                  <PaginationItem>
-                    <PaginationLink
-                      className="text-info"
-                      previous
-                      onClick={() => this.navigatePage(page - 1)}
-                    />
-                  </PaginationItem>
-                )}
-                {pageNums.map(num => (
-                  <PaginationItem
-                    key={num}
-                    active={num === page}
-                    className="text-info pagination-active"
-                  >
-                    <PaginationLink
-                      className="text-info"
-                      onClick={() => this.navigatePage(num)}
-                    >
-                      {num}
-                    </PaginationLink>
-                  </PaginationItem>
-                ))}
-                {page < count && (
-                  <PaginationItem>
-                    <PaginationLink
-                      className="text-info"
-                      next
-                      onClick={() => this.navigatePage(page + 1)}
-                    />
-                  </PaginationItem>
-                )}
-              </Pagination>
-            </Row>
-          )}
+          <div>
```

Comment:
I'm curious why you added a `div` here.

Reply:
Oh! I am accustommed of doing it to avoid error of react component should be enclosed inside a parent element.Will remove it in a while

Reply:
You can use `React.Fragment` for that :)